### ISSUE TITLE
Issues with autocomplete

### ISSUE THREAD

**Issue by Trid-collab** [bpython/bpython#897]
Hi, I am new bpython. I am using 0.21.
 I found though the Autocomplete worked well in most cases like the following I had problems . When I type say "from openpyxl import Workbook, load_workbook" the autcomplete for the imports Workbook, load_workbook does'nt work. 
Also when type "import matplotlib.pylot as plt " the autocomplete after the "." import matplotlib.  doesn't work. 
Am I missing something here or this is how it is supposed to work

**Comment by sebastinas** [comment]
Indeed, there are some cases that `bpython` currently cannot handle. See https://github.com/bpython/bpython/blob/main/bpython/test/test_importcompletion.py#L50 for some of them. If you have a lot of package installed, import completion may take some time to find them.
